### ISSUE TITLE
TwoStageReduce.py AttributeError: 'Word2Vec' object has no attribute 'vocab'

### ISSUE THREAD

**Issue by washingon** [jeffThompson/Word2VecAndTsne#4]
Jeff - I get error AttributeError: 'Word2Vec' object has no attribute 'vocab' in TwoStageReduce.py

Environment:  Ubuntu 16.04.3 LTS  python 2.7   gensim (3.2.0)
Steps to reproduce error:

1. modify TrainModel.py line 38 to: model.train(sentences, total_examples=model.corpus_count, epochs=model.iter) as noted in #3 
2. python TrainModel.py
3. python TwoStageReduce.py

```
loading model...
- done
converting model/words to numpy array...
- loading 50000 most common words...
- creating list of words/vectors for reduction...
  - keeping only 50000 most common words
Traceback (most recent call last):
  File "TwoStageReduce.py", line 99, in <module>
    for word in model.vocab:
AttributeError: 'Word2Vec' object has no attribute 'vocab'

```

**Comment by jeffThompson** [comment]
Hmm, I think it's due to a change in gensim. [This post](https://stackoverflow.com/a/35641434/1167783) seems to suggest you have to change the syntax for accessing the `vocab`.

Can you try changing the line to `for word in model.wv.vocab:` and see if it works?

**Comment by washingon** [comment]
yes, I can correct the error by changing line 99 in TwoStageReduce.py
from:  for word in model.vocab:
to:  for word in model.wv.vocab:

**Comment by jeffThompson** [comment]
Great! Please let me know if you hit any other errors and I'll update the repo.

**Comment by kusumlata123** [comment]
if word in word2vec_model.wv.vocab:
TypeError: unhashable type: 'list'
